nx/include/switch/services/set.h

```diff
@@ -1017,23 +1028,269 @@ void setcalExit(void);
 /// Gets the Service object for the actual setcal service session.
 Service* setcalGetServiceSession(void);
 
-/// Gets the \ref SetCalEccB233DeviceCertificate.
+/**
+ * @brief Gets the \ref SetCalBdAddress.
+ * @param[out] out \ref SetCalBdAddress
+ */
+Result setcalGetBdAddress(SetCalBdAddress *out);
+
+/**
+ * @brief Gets the \ref SetCalConfigurationId1.
+ * @param[out] out \ref SetCalConfigurationId1
+ */
+Result setcalGetConfigurationId1(SetCalConfigurationId1 *out);
+
+/**
+ * @brief Gets the \ref SetCalAccelerometerOffset.
+ * @param[out] out \ref SetCalAccelerometerOffset
+ */
+Result setcalGetAccelerometerOffset(SetCalAccelerometerOffset *out);
+
+/**
+ * @brief Gets the \ref SetCalAccelerometerScale.
+ * @param[out] out \ref SetCalAccelerometerScale
+ */
+Result setcalGetAccelerometerScale(SetCalAccelerometerScale *out);
+
+/**
+ * @brief Gets the \ref SetCalAccelerometerOffset.
+ * @param[out] out \ref SetCalAccelerometerOffset
+ */
+Result setcalGetGyroscopeOffset(SetCalAccelerometerOffset *out);
+
+/**
+ * @brief Gets the \ref SetCalGyroscopeScale.
+ * @param[out] out \ref SetCalGyroscopeScale
+ */
+Result setcalGetGyroscopeScale(SetCalGyroscopeScale *out);
+
+/**
+ * @brief Gets the \ref SetCalMacAddress.
+ * @param[out] out \ref SetCalMacAddress
+ */
+Result setcalGetWirelessLanMacAddress(SetCalMacAddress *out);
+
+/**
+ * @brief GetWirelessLanCountryCodeCount
+ * @param[out] out_count Output count
+ */
+Result setcalGetWirelessLanCountryCodeCount(s32 *out_count);
+
+/**
+ * @brief GetWirelessLanCountryCodes
+ * @param[out] total_out Total output entries.
+ * @param[out] codes Output array of \ref SetCalCountryCode.
+ * @param[in] count Size of the versions array in entries.
+ */
+Result setcalGetWirelessLanCountryCodes(s32 *total_out, SetCalCountryCode *codes, s32 count);
+
+/**
+ * @brief Gets the \ref SetCalSerialNumber.
+ * @param[out] out \ref SetCalSerialNumber
+ */
+Result setcalGetSerialNumber(SetCalSerialNumber *out);
+
+/**
+ * @brief SetInitialSystemAppletProgramId
+ * @param[in] program_id input ProgramId.
+ */
+Result setcalSetInitialSystemAppletProgramId(const u64 *program_id);
```

Comment:
Don't use ptrs for input u64s when it's not arrays (same elsewhere).

Reply:
Fixed.